apps/total-typescript/src/pages/invoices/[merchantChargeId].tsx

```diff
@@ -158,9 +161,9 @@ const Invoice: React.FC<
                   Invoice For
                 </h2>
                 <div>
-                  {charge.billing_details.name}
+                  {customer.name}
```

Comment:
billing details are at time of purchase so using customer here because it can update